### PR TITLE
Add basic TM and public IP

### LINKED ISSUE #18: Infrastructure: Direct traffic to AKS cluster and services
Currently, there is no way of directing traffic into our AKS cluster. In order to achieve this, we need a traffic manager that directs traffic to the AKS cluster's public IP resource.

### Acceptance Criteria
- [x] Should have a traffic manager provisioned

### PR DESCRIPTION
**Pull request by adelechurchill**
Basic Traffic Manger and Public IP

### CODE REVIEW

**Review by decodingahmed — APPROVED** [review]

File: infrastructure/modules/resource_templates/traffic_manager/main.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    protocol                     = "HTTP"
+    port                         = 80
+    path                         = "/"
+    interval_in_seconds          = 30
+    timeout_in_seconds           = 9
+    tolerated_number_of_failures = 3
+  }
+
+  tags = {
+    environment = "Production"
```
Comment (guypritchard) [inline infrastructure/modules/resource_templates/traffic_manager/main.tf:32]: Does this mean this is the legit 'Production' environment or something else?  (Tell me if I'm being dumb)

**Review by guypritchard — APPROVED** [review]

File: infrastructure/main.tf [hunk 2]
```diff
@@ -43,3 +47,11 @@ module "aks" {
   resource_group_name = module.resource_group.resource_group_name
   depends_on          = [module.resource_group]
 }
+
+module "traffic_manager" {
+  source              = "./modules/resource_templates/traffic_manager"
+  name                = local.traffic_manager_name
+  resource_group_name = module.resource_group.resource_group_name
+  public_ip_name      = local.public_ip_name
+  depends_on          = [module.resource_group]
```
Comment (yahya130) [inline infrastructure/main.tf:56]: Just a thought, but would it be useful making this module dependent on the AKS module being deployed? that way we could tie the two together and ensuring a traffic manager would only be there if the AKS cluster is?

**Review by yahya130 — APPROVED** [review]

File: infrastructure/modules/resource_templates/traffic_manager/main.tf (hunk 1 above)
Comment (decodingahmed) [reply infrastructure/modules/resource_templates/traffic_manager/main.tf:32]: @guypritchard - this is handled in the next PR that @adelechurchill and I have worked on. See this: https://github.com/buerokratt/Infrastructure/blob/8b5594ce3fd0d0d9ac25c057b35a74cf719395ce/infrastructure/modules/traffic_manager/main.tf#L25

**Comment by adelechurchill** [comment]
Closed as this change is addressed in https://github.com/buerokratt/Infrastructure/pull/25

**Pull request closed without merging**